SPECS/ocaml-augeas/ocaml-augeas.spec

```diff
@@ -5,24 +5,23 @@ ExcludeArch: %{ix86}
 Summary:        OCaml bindings for Augeas configuration API
 Name:           ocaml-%{srcname}
 Version:        0.6
-Release:        32%{?dist}
+Release:        33%{?dist}
 License:        LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://people.redhat.com/~rjones/augeas/files/
 Source0:        https://people.redhat.com/~rjones/%{srcname}/files/%{name}-%{version}.tar.gz
-Source1:        ocaml_files.py
-Source2:        macros.ocaml-rpm
```

Comment:
You removed the `Source` tags but I think you missed removing the files themselves.

Reply:
Fixed this